docs/authentication.md

```diff
@@ -130,6 +130,17 @@ Both of these libraries support either authentication pattern. If you're interes
 - [with-passport](https://github.com/vercel/next.js/tree/canary/examples/with-passport)
 - [with-passport-and-next-connect](https://github.com/vercel/next.js/tree/canary/examples/with-passport-and-next-connect)
 
+### SuperTokens
```

Comment:
Could you move this down to the bottom of the file please? I don't want to show preference by adding a new provider at the top.

Reply:
Of course that makes sense! Done now :)